File: engine/model/src/main/scala/tools/packet/remove/RemovePlayer.scala
package tools.packet.remove

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemovePlayer(private var _characterId: Int) extends PacketInput {
  def characterId: Int = _characterId

  override def opcode(): SendOpcode = SendOpcode.REMOVE_PLAYER_FROM_MAP
}

File: engine/engine-model/src/main/java/tools/packet/remove/RemovePlayer.java
package tools.packet.remove;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemovePlayer(Integer characterId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.REMOVE_PLAYER_FROM_MAP;
   }
}